mx.sulong/mx_sulong.py

```diff
@@ -573,10 +573,9 @@ def runLLVMTestCases(args=None):
 
 def runTruffleTestCases(args=None):
     """runs the Sulong test suite"""
-    ensureLLVMBinariesExist()
-    ensureDragonEggExists()
+    testsuites.compileSuite(['sulong'])
     vmArgs, _ = truffle_extract_VM_args(args)
-    return unittest(getCommonUnitTestOptions() + ['-Dsulong.ExecutionCount=1000'] + vmArgs + ["com.oracle.truffle.llvm.test.SulongTestSuite"])
+    return unittest(vmArgs + ["com.oracle.truffle.llvm.test.alpha.SulongSuite"])
```

Comment:
If we remove the execution count, we do not longer test that Graal can still compile our nodes. This is a problem, as long as we do not integrate such a downstream test.